internal/sim/sim.go

```diff
@@ -273,6 +279,10 @@ func New(name string, opts Options) (*Simulator, error) {
 	// Create simulator.
 	//
 	// TODO(mwhittaker): Take a *testing.T and use the test name as the name.
+	notFinished := map[int]struct{}{}
+	for i := 1; i < opts.NumOps+1; i++ {
+		notFinished[i] = struct{}{}
```

Comment:
Any way to replace `i` with `traceID`, to make it clearer? This is also assuming that trace ids map to ops 1-1. Also, you use the term "op id" in other places.